src/lib/components/tooltips/Tooltip.svelte

```
@@ -0,0 +1,37 @@
+<script lang="ts">
+	export let text: string;
+	export let pos = {
+		x: 0,
+		y: 0
+	};
+	export let opacity = 0;
+
+	let toolTipHeight: number;
+	let toolTipWidth: number;
+</script>
+
+<div
+	style="left: {pos.x - toolTipWidth / 2}px; top: {pos.y - toolTipHeight / 4}px; opacity: {opacity}"
+	class="wrap"
+>
+	{#if text}
```

why is this hidden?

---

if this is *in case* the text doesnt exist, its better to fail fast here